README.md

```diff
@@ -18,6 +18,8 @@ This is a community project so feel free to contribute ideas, submit bugs, fix b
 
 To use the GCM, you can download the [latest installer](<https://github.com/Microsoft/Git-Credential-Manager-for-Windows/releases/latest>). To install, double-click Setup.exe and follow the instructions presented.
 
+When prompted to select your terminal emulator for Git Bash you should choose the Windows' default console window. GCM will not prompt you for credentials in a MinTTY setup.
```

Comment:
But this should be made clear in the readme, because currently GCM requires you to change the git installation defaults to work properly, but does document anywhere that it requires this.

Hence the suggested change to the readme.